src/collection-browser.ts

```diff
@@ -376,6 +381,8 @@ export class CollectionBrowser
         .displayMode=${this.displayMode}
         .selectedTitleFilter=${this.selectedTitleFilter}
         .selectedCreatorFilter=${this.selectedCreatorFilter}
+        .titleLetterCounts=${this.titleLetterCounts}
```

Comment:
Question: Do we really need counts for both title and creator? Would a single `.charCounts` var work?

Reply:
My thinking around this was, since generating aggregation data tends to be rather expensive on the backend, we should only (lazily) request what we actually need to avoid unnecessary overhead, and should cache the counts to avoid having to recompute them if users switch between title/creator sorting. Having them separate avoids the need to repeatedly request & overwrite the same data when people change the sort type.

That said, maybe I'm just prematurely optimizing... it's possible the `firstX` counts aren't actually as expensive as other aggregation types.

Reply:
>avoid having to recompute them if users switch between title/creator sorting
Thanks @latonv. This makes sense to me.

Reply:
Will resolve relevant conversations.